AngouriMath/Functions/Continuous/Algebra/Integration.cs

```diff
@@ -53,7 +53,7 @@ namespace AngouriMath
 }
 namespace AngouriMath.Functions.Algebra
 {
-    internal static class Integration
+    public static partial class Integration
```

Comment:
Better to keep this internal. Should be consistent with Limit, so that you expose it only in MathS.Compute